packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -2525,6 +2579,54 @@ class RenderPointerListener extends RenderProxyBoxWithHitTestBehavior {
   /// no longer directed towards this receiver.
   PointerCancelEventListener onPointerCancel;
 
+  // Object used for annotation of the layer used for hover hit detection.
+  MouseTrackerAnnotation _hoverAnnotation;
+
+  void _updateAnnotations() {
+    if (_hoverAnnotation != null) {
+      RendererBinding.instance.mouseTracker.detachAnnotation(_hoverAnnotation);
```

Comment:
Do we need to guard this call to only happen when render object is attached?

Reply:
Yes, I guess that would be prudent.